components/database/mongodb-40/Makefile

```diff
@@ -104,35 +132,37 @@ COMPONENT_TEST_TRANSFORMS +=    '-e "s/\[resmoke\][ -+0-9T:\.\-]*//"' \
                                 '-n ' \
                                 '-e "/Summary of all suites/,\\$$p"'
 
-
 # Not strictly necessary
 $(TEST_64):     $(INSTALL_64) $(INSTALL_TOOLS)
 
 build:		$(BUILD_64) $(BUILD_TOOLS)
 
-install:	$(INSTALL_64) $(INSTALL_TOOLS)
+install: $(INSTALL_TOOLS)
+	$(MKDIR) $(PROTO_DIR)$(MONGO_PREFIX)/man/man1
+	$(CP) $(BUILD_DIR_64)/debian/*.1  $(PROTO_DIR)$(MONGO_PREFIX)/man/man1/
+	$(CP) $(BUILD_DIR_64)/debian/mongo{,d,s,ldap,kerberos}.1  $(PROTO_DIR)$(MONGO_PREFIX)/man/man1/
 
 test:		$(TEST_64)
 
 # Test requirements
-REQUIRED_PACKAGES += library/python/pymongo-27
-REQUIRED_PACKAGES += library/python/pyyaml
-REQUIRED_PACKAGES += library/python/urllib3-27
+#REQUIRED_PACKAGES += library/python/pymongo-39
+#REQUIRED_PACKAGES += library/python/urllib3-39
```

Comment:
Why are these required packages commented out?

Reply:
Only needed for testing.
If you would like to test you have to change the target to "install-all". Compilation of the package will last several hours before the testing!